src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -27,6 +27,7 @@
   public static final String ENTITY_KIND = "Candidate";
   public static final String NAME_JSON_KEYWORD = "name";
   public static final String PARTY_JSON_KEYWORD = "party";
+  public static final String CAMPAIGN_JSON_KEYWORD = "candidateUrl";
```

Comment:
Prefer more descriptive variable names, especially in java land. 

Change this to CANDIDATE_CAMPAIGN_URL_JSON_KEYWORD. CAMPAIGN_JSON_KEYWORD makes me do a double take when I see that this is actually about a candidate's URL.

Reply:
done